sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -538,31 +538,22 @@ Text Analytics for health is a containerized service that extracts and labels re
 ```
 
 ### Run multiple actions Asynchronously
-This functionality allows running multiple actions in one or more documents. Actions include entity recognition, linked entity recognition, key phrase extraction, and Personally Identifiable Information (PII) Recognition. For more information see [Using analyze][analyze_operation_howto].
+This functionality allows running multiple actions in one or more documents. Actions include entity recognition, linked entity recognition, key phrase extraction, Personally Identifiable Information (PII) Recognition, and sentiment analysis. For more information see [Using analyze][analyze_operation_howto].
```

Comment:
nit: captalize sentiment analysis

Reply:
should I capitalize all actions too? not sure what is right in English :P 
I did it for PII as it is a known concept

Reply:
ah ok that makes sense. ignore me then